src/common/TableOfContents.css

```diff
@@ -0,0 +1,63 @@
+/* FAQ Table of Contents */
+.toc-container {
+  position: sticky !important;
+  position: -webkit-sticky; /* For Safari */
+  margin-top: 10px;
+  padding-top: 5px;
+  max-height: calc(100vh - 40px);
+  top: 0;
+  width: 250px;
+  margin-right: 5px;
+  margin-left: 0px;
+  overflow: auto;
+}
+
+.initial-header {
+  margin-top: 40px;
+}
+
+.initial-header li {
+  list-style-type: none;
+}
+
+.initial-header li a {
+  text-decoration: none;
+  color: black;
+}
+
+.highlight {
+  border-style: solid;
+  border-left-color:#0071bc;
+  border-left-width: 4px;
+  border-bottom: none;
+  border-top: none;
+  border-right: none;
+  padding-left: 4px;
+  transition: all 1s ease-in;
+}
+
+.header {
+  font-size: 16px;
+  font-weight: 800;
+}
+
+.subheader {
+  font-size: 14px;
+  padding-left: 20px;
+  padding-right: 20px;
+  font-weight: normal;
+  text-overflow: ellipsis;
+
+  /* Required for text-overflow to do anything */
+  /* white-space: nowrap; */
+  /* overflow: hidden; */
```

Comment:
Probably not needed anymore. This would cutoff the verbiage from the h2/h3 in the side bar ensuring the text didn't encroach more then one line.

Removed commented code.